src/create-http-client.ts

```diff
@@ -0,0 +1,41 @@
+import path from 'path';
+import fs from 'fs';
+import https from 'https';
+import axios, { AxiosInstance, AxiosRequestConfig } from 'axios';
+import { cloneDeep } from 'lodash';
+
+import { version } from '../package.json';
+
+export interface IMollieRequestConfig extends AxiosRequestConfig {
+  apiKey: string;
+}
+
+declare let window: any;
+
+/**
+ * Create pre-configured httpClient instance
+ *
+ * @private
+ */
+export default function createHttpClient(options: IMollieRequestConfig): AxiosInstance {
+  const newOptions = cloneDeep(options);
+  newOptions.baseURL = 'https://api.mollie.com:443/v2/';
+
+  newOptions.headers = {
+    ...options.headers,
+    Authorization: `Bearer ${options.apiKey}`,
+    'Accept-Encoding': 'gzip',
+    'Content-Type': 'application/json',
+    'User-Agent': `node.js/${process.version}`,
+    'X-Mollie-User-Agent': `mollie/${version}`,
```

Comment:
I was checking our backend to see the metrics for the Node 2.x client but couldn't find any. 

I found the problem. We are only parsing the `User-Agent` header, `X-Mollie-User-Agent` is ignored. We currently also do not recognize `node.js` or `mollie`, only `Node` and `Mollie`.

Can you please change the user agent header to `Node/${process.version} Mollie/${version}`?

Reply:
Sure, I have updated the `User-Agent`.